techniques/aria/ARIA12.html

```diff
@@ -33,41 +41,55 @@ <h3>Additional heading levels</h3>
 &lt;h6&gt;Apples&lt;/h6&gt;
 &lt;p&gt;Apples grow on trees in areas known as orchards...&lt;/p&gt;
    ...
-&lt;div role="heading" aria-level="7"&gt;Jonagold/div&gt;
+&lt;div role="heading" aria-level="7"&gt;Jonagold&lt;/div&gt;
 &lt;p&gt;Jonagold is a cross between the Golden Delicious and Jonathan varieties...&lt;/p&gt;</code></pre>
-         
-      </section>
-   </section><section id="tests"><h2>Tests</h2>
-      <section class="procedure"><h3>Procedure</h3>
-         <ol>
-            <li> Examine each element with the attribute <code class="language-html">role="heading"</code>.
-</li>
-            <li> Determine whether the content of the element is appropriate as a heading.
-</li>
-            <li> If the element has an <code class="language-html">aria-level</code> attribute, determine whether the value is the appropriate hierarchical level.
-</li>
-         </ol>
+	<aside class="note">
+		<p>Headings with an <code class="language-html">aria-level</code> of 10 or higher can create difficulties for users. At the time of writing, most combinations of user agents and assistive technologies report such levels as being level 2.</p>
```

Comment:
That seems really odd to me. Is there a reference?

Reply:
I mentioned it in the issue—I found it in the [ARIA 1.3 draft](https://www.w3.org/TR/wai-aria-1.3/#aria-level) and tweaked it a bit.